benchmarks/dhrystone/CMakeLists.txt

```diff
@@ -44,7 +44,10 @@ if(CONFIG_BENCHMARK_DHRYSTONE)
     endif()
   endif()
 
-  set(CFLAGS -DMSC_CLOCK)
+  set(CFLAGS
```

Comment:
Wouldn't it be better to fix the code than disable warnings? :-)

Reply:
dhrystone is the 3rd party library, no critical change should upstream to the official project, so I would suggest you push the change to them, and we can update the download to new version.

Reply:
Since the different project has different warning setting. It's useless to fix the warning message or even wrong, before the fix gets accepted by the project maintainer.

Reply:
Not really, we already patch the release code `malloc` and boolean statements, so we modify the code anyway, we can also patch the uninitialized variables. Instead we want to disable build warnings that may hide potential problems in future.

This kind of patching is common also in FreeBSD Ports. Then patches are supposed to land in the upstream so they are not needed in future releases :-)

I can see some flags were present before, so okay, we may include them :-) I would go for patching the upstream too :-)

Reply:
we made this modification because there is no way to disable it from -Wno-xxx option or it's too dangerous to disable it. For the minor warning, it's better to avoid patching the code before the change upstreamed.